docs/man/ctags-lang-python.7.rst

```diff
@@ -0,0 +1,167 @@
+.. _ctags-lang-python(7):
+
+==============================================================
+ctags-lang-python
+==============================================================
+-------------------------------------------------------------------
+Random notes about tagging python source code with Universal-ctags
+-------------------------------------------------------------------
+:Version: 0.0.0
+:Manual group: Universal-ctags
+:Manual section: 1
+
+SYNOPSIS
+--------
+|	**ctags** ... --languages=+Python ...
+|	**ctags** ... --language-force=Python ...
+|	**ctags** ... --map-Python=+.py ...
+
+DESCRIPTION
+-----------
+This man page gathers random notes about tagging python source code.
+
+EXAMPLES
+--------
+This section shows how ctags uses kinds, roles, fields, and extras when tagging
+python source code for given input.
+
+Tagging ``import`` statements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+"input.py"
+.. code-block:: Python
+
+   import X0
+
+"output.tags"
+with "--options=NONE -o - --extras=+r --fields=+rzK input.py"
+.. code-block:: tags
+
+	X0	input.py	/^import X0$/;"	kind:module	roles:imported
+
+A tag for an imported module has ``module`` kind with ``imported`` role.  The
+module is not defined here; it is defined in another file. So the tag for the
+imported module is a reference tag; specify ``--extras=+r`` (or
+``--extras=+{reference}``) option for tagging it.  "roles:" field enabled with
+``--fields=+r`` is for recording the module is "imported" to the tag file.
+
+"input.py"
+.. code-block:: Python
+
+	import X1 as Y1
+
+"output.tags"
+with "--options=NONE -o - --extras=+r --fields=+rzK --fields-Python='+{nameref}' input.py"
+::
+
+	X1	input.py	/^import X1 as Y1$/;"	kind:module	roles:indirectlyImported
+	Y1	input.py	/^import X1 as Y1$/;"	kind:namespace	roles:def	nameref:module:X1
+
+"Y1" introduces a new name and is defined here. So "Y1" is tagged as a
+definition tag.  "X1" is imported in the way that its name cannot be used
+here. For letting know client tools that the name cannot be used,
+``indirectlyImported`` role is assigned for "X1".  "Y1" is the name for
+accessing objects defined in the module imported via "X1".  For recording this
+relationship, ``nameref:`` field is attached to the tag of "Y1".  Instead of
+``module`` kind, ``namespace`` kind is assigned to "Y1" because "Y1" itself
+doesn't define a module.
```

Comment:
```suggestion
isn't a module.
```

I think "doesn't define a module" is a bit ambiguous, since this is a reference tag, and the module is not defined here anyway.